frontend/packages/dev-console/src/components/pipelines/resource-overview/TriggerTemplateResourceLink.tsx

```diff
@@ -34,11 +34,6 @@ const TriggerTemplateResourceLink: React.FC<TriggerTemplateResourceLinkProps> =
                 title={triggerTemplateName}
                 inline
               />
-              {routeURL && (
```

Comment:
This will remove the link from the Pipeline Details page - which is undesirable.

Reply:
I think that cleared my confusion. Please take another look.